config/translations/en/module.download.yml

```diff
@@ -3,6 +3,9 @@ arguments:
     module: 'Module or modules to be enabled should be separated by a space'
 options:
     latest: 'Default to download most recent version'
+    module-path: 'The path of the contrib project'
```

Comment:
@Saphyel , That is minor improvement that we recently started to do whenever we do new enhancement, which is cleaning the redundancy between names.

Hopefully for the stable release we could have all command with clean names.